src/image.rs

```diff
@@ -50,7 +50,7 @@ pub struct Image {
     pub depth: u8,
 
     pub yuv_format: PixelFormat,
-    pub full_range: bool,
+    pub full_range: bool, // VideoFullRangeFlag as specified in ISO/IEC 23091-4/ITU-T H.273.
```

Comment:
Is the ISO/IEC standard number -4 or -2?

-4 is used here at at line 58, and -2 is used at 103. According to https://www.itu.int/rec/T-REC-H.273-201612-S/en, -2 is the right one.

Reply:
https://aomediacodec.github.io/av1-spec/#color-config-semantics says

> **color_range** is a binary value that is associated with the VideoFullRangeFlag variable specified in ISO/IEC 23091-4/ITU-T H.273.

so I used both.

Reply:
Thanks. That is very likely an error in the AV1 spec. I have both ISO/IEC 23091-2:2019 and ISO/IEC 23091-2:2021. I can confirm ISO/IEC 23091-2 is the equivalent of ITU-T H.273 and `VideoFullRangeFlag` is specified in ISO/IEC 23091-2.

Reply:
I filed an issue with the AV1 spec: https://github.com/AOMediaCodec/av1-spec/issues/345.

Reply:
I filed a PR with the AV1 spec: https://github.com/AOMediaCodec/av1-spec/pull/344